app/src/App/App.css

```diff
@@ -85,7 +85,8 @@ fieldset[disabled] .gatekeeper-btn.active {
 
 /* antd overrides */
 
-.custom-ant-spin {
+.custom-ant-spin,
+.ant-spin {
```

Comment:
I think we need to apply the custom class name to spin components instead of targeting all of them with the default antd className

Reply:
I would be ok, doing this in its own pr

Reply:
cool will create an issue to track this

Reply:
https://github.com/OpenSRP/web/issues/537